examples/networking/router_route/base-network.tf

```diff
@@ -0,0 +1 @@
+../network/main.tf
```

Comment:
we can use entire networking/main example for router_route

Reply:
yes used base-networking instead of small links